lib/start/src/bottom_stuff.dart

```diff
@@ -94,7 +105,7 @@ class BottomStuff extends HookWidget {
           return Padding(
             padding: EdgeInsets.only(top: 20 * tSeparator),
             child: SizedBox(
-              height: 88 * tSeparator,
+              height: math.max(88 * tSeparator, 0.0),
```

Comment:
Same thing here as above: please let me know if removing the `math.max()` call creates any errors, since that would be worth investigating.